completions/bash/swaymsg

```diff
@@ -14,7 +14,8 @@ _swaymsg()
     'get_marks'
     'get_bar_config'
     'get_version'
-    'get_clipboard'
+	'get_binding_modes'
+	'get_config'
```

Comment:
This file uses two spaces for indenting but these lines are using tabs. I prefer tabs, but let's change these to spaces for consistency.